README.md

```diff
@@ -41,3 +41,19 @@ Refer to [RFC2119](https://www.ietf.org/rfc/rfc2119) for details.
 
 _Please contact_ [_Zdenek.Nemec@externals.adidas-group.com_](mailto:Zdenek.Nemec@externals.adidas-group.com) _in the case of questions._
 
+## Intended Use Cases
+This project is intended to provide the guidelines for design & development of APIs at adidas.
+
+adidas is not responsible for the usage of this software for different purposes that the ones described in the use cases.
+
+## License and Software Information
+ 
+© adidas AG
+
+adidas AG publishes this software and accompanied documentation (if any) subject to the terms of the MIT license with the aim of helping the community with our tools and libraries which we think can be also useful for other people. You will find a copy of the MIT license in the root folder of this package. All rights not explicitly granted to you under the MIT license remain the sole and exclusive property of adidas AG.
+ 
+NOTICE: The software has been designed solely for the purpose of providing API design and development guidelines. The software is NOT designed, tested or verified for productive use whatsoever, nor or for any use related to high risk environments, such as health care, highly or fully autonomous driving, power plants, or other critical infrastructures or services.
+ 
+If you want to contact adidas regarding the software, you can mail us at _software.engineering@adidas.com_.
+ 
+For further information open the [adidas terms and conditions](https://github.com/adidas/adidas-contribution-guidelines/wiki/Terms-and-conditions) page.
```

Comment:
It is a good idea to have the `CONTRIBUTING.md` and the `CODE_OF_CONDUCT.md` files.

You can see an example:

- [`CONTRIBUTING.md`](https://github.com/adidas/js-linter-configs/blob/master/.github/CONTRIBUTING.md).
- [`CODE_OF_CONDUCT.md`](https://github.com/adidas/js-linter-configs/blob/master/CODE_OF_CONDUCT.md).

Reply:
Is this needed? If so please add it to the confluence required steps. If not, please let's move forward.

Reply:
It is not really mandatory, but if you want someone else to contribute to the repository is a good idea to show the contribution guidelines.

All this useful information is in the _Good practices for public code_ section.

Reply:
Added.